staging/src/k8s.io/code-generator/cmd/go-to-protobuf/protobuf/generator.go

```diff
@@ -399,7 +399,7 @@ func (b bodyGen) doStruct(sw *generator.SnippetWriter) error {
 				}
 				extras = append(extras, fmt.Sprintf("%s = %s", k, v))
 			}
-			sort.Sort(sort.StringSlice(extras))
+			sort.Strings(extras))
```

Comment:
There is no point in retesting - this line doesn't compile (too many ")" at the end.

Reply:
My bad, thanks for pointing out. Changes done.